lib/index.js

```diff
@@ -1,3 +1,5 @@
+// Foods
```

Comment:
Fine for now- but let's make sure we break this down at least two js files for food and meals